### PR TITLE
Metazoan species renamed for RR40

### PR DESCRIPTION
**Pull request by marcoooo**
## Description

Following updates in many metazoan species names / URL and display Name. Related updated in compara-configuration for homologies configuration run.

**Related JIRA tickets:**
- ENSRR-449

## Overview of changes
Updates in species production names. 

#### Change 1
New names in conf/references/mlss_conf.xml

#### Change 2
New names in conf/references/allowed_species.json

## Notes
BTw, I hope this doesn't need a complete Reference rerun? Does it? 
---

## PR review checklist

- [ ] Is the PR against an appropriate branch?
- [ ] Does the code adhere to coding guidelines?
- [ ] Does the code do what it claims to do?
- [ ] Is the code readable? Is it appropriately documented?
- [ ] Is the logic in the correct place?
- [ ] Was the code tested appropriately? Are there unit tests? Are unit tests self-contained and non-redundant?
- [ ] Did Travis CI pass for the code in the PR? Is Codecov acceptable based on the included/updated unit tests?
- [ ] Will the new code fail gracefully?
- [ ] Does the code follow good practice for writing performant code (e.g. using a database transaction rather than repeated queries outside of a transaction)?
- [ ] Does it bring in an unnecessary dependency?
- [ ] If you are reviewing a new analysis, is it future-proof and pluggable?
- [ ] Does the PR meet agile guidelines?

### CODE REVIEW

**Comment by ens-sb** [comment]
@marcoooo Another repeat: "cimex_lectularius_gca000648675v3rs_gca000648675v3rs_..."

File: conf/references/allowed_species.json [hunk 1]
```diff
@@ -44,35 +44,35 @@
...
-    "dendroctonus_ponderosae_gca000355655v1",
+    "dendroctonus_ponderosae_gca000355655v1rs",
     "denticeps_clupeoides",
-    "diabrotica_virgifera_gca003013835v2",
+    "diabrotica_virgifera_gca003013835v2rs",
     "dicentrarchus_labrax",
     "dictyostelium_discoideum",
     "dipodomys_ordii",
-    "drosophila_melanogaster",
+    "drosophila_melanogaster_gca000001215v4fb",
```
Comment (vsitnik) [inline conf/references/allowed_species.json]: decided not to change

File: conf/references/mlss_conf.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
       <genome name="gallus_gallus"/>
       <genome name="danio_rerio"/>
       <!-- Metazoa -->
-      <genome name="drosophila_melanogaster"/>
+      <genome name="drosophila_melanogaster_gca000001215v4fb"/>
```
Comment (vsitnik) [inline conf/references/mlss_conf.xml]: decided not to change

File: conf/references/allowed_species.json (hunk 1 above)
Comment (marcoooo) [reply conf/references/allowed_species.json]: Sorry, I just reverted. Thanks!

**Comment by marcoooo** [comment]
> cimex_lectularius_gca000648675v3rs_gca

Couldn't find it, did you fix it?

**Comment by vsitnik** [comment]
@twalsh-ebi @ens-LCampbell could you take a look as well? Thanks!

**Review by ens-LCampbell — APPROVED** [review]
Addition of suffix changes looks good. Happy for PR merge

**Comment by twalsh-ebi** [comment]
Hi @marcoooo.. Travis is complaining about unit-test databases. There were some big changes to test databases in the recent Compara commits [63b4bc3](https://github.com/Ensembl/ensembl-compara/commit/63b4bc3bbb575b84084c94c596702faeb53ff67e) and [783e7f3](https://github.com/Ensembl/ensembl-compara/commit/783e7f3b6d7e95435a858527492b46932a6e82d4) on the `feature/homology_annotation` branch. Would it be possible to rebase `update-metazoa-names` on `feature/homology_annotation`?

**Comment by marcoooo** [comment]
> Hi @marcoooo.. Travis is complaining about unit-test databases. There were some big changes to test databases in the recent Compara commits [63b4bc3](https://github.com/Ensembl/ensembl-compara/commit/63b4bc3bbb575b84084c94c596702faeb53ff67e) and [783e7f3](https://github.com/Ensembl/ensembl-compara/commit/783e7f3b6d7e95435a858527492b46932a6e82d4) on the `feature/homology_annotation` branch. Would it be possible to rebase `update-metazoa-names` on `feature/homology_annotation`?

Hi @twalsh-ebi , I created my branch from this one, not sure what to do to help.

**Comment by ens-sb** [comment]
@marcoooo Did you create the `update-metazoa-names` branch locally? If yes what you might need is to do a `git pull origin feature/homology-annotation`.

**Comment by twalsh-ebi** [comment]
Thanks @marcoooo.. It turns out there may be an issue with the `feature/homology_annotation` branch itself. We will troubleshoot that and get back to you.

**Review by ens-sb — APPROVED** [review]

**Review by twalsh-ebi — APPROVED** [review]

**Pull request merged**